docs/source/api_reference/data_format.rst

```diff
@@ -0,0 +1,248 @@
+Each identifier must be present at a separate line.
+The format of string identifiers are: ``@<identifier> [value]``, except for ``@data`` where there is no trailing information.
+
+.. note::
+    Since these datasets are often from different sources (see `tsregression`_ and `timeseriesclassification.com`_)
+    There may be minor conflict in their naming conventions (lowercase vs. camelCase). ``sktime`` internally takes care of such inconsistencies.
+
+    For this document, we will only use lowercase to represent the identifier.
+
+    However, if you run into an inconsistency that isn't already taken care of, kindly consider opening an `issue`_.
+
+.. list-table::
+    :widths: 10 40 30 20
+    :header-rows: 1
+
+    * - Identifier
+      - Description
+      - Additional Comments
+      - Example
+    * - ``@problemname``
+      - The name of the dataset.
+      - Value cannot be space seperated
+      - ``BasicMotions``
+    * - ``@timestamps``
+      - Whether timestamps are present.
+      - ``true`` / ``false`` only
+      - ``false``
+    * - ``@missing``
+      - Whether there are missing values.
+      - ``true`` / ``false`` only
+      - ``false``
+    * - ``@univariate``
+      - Whether there is only one variable.
+      - ``true`` / ``false`` only
+      - ``false``
+    * - ``@dimension``
+      - The number of variables.
+      - Only present when ``@univariate=false``.
+      - 6
+    * - ``@equallength``
+      - Whether each instance has equal length.
+      - ``true`` / ``false`` only
+      - ``true``
+    * - ``@serieslength``
+      - Number of timestamps in each instance.
+      - Only present if ``@equallength=true``.
+      - 100
+    * - ``@targetlabel``
+      - Whether there is a target label.
+      - Exclusive to regression data; ``true`` / ``false`` only
+      - ``true``
+    * - ``@classlabel``
+      - Whether class labels are present.
+      - Exclusive to classification data; when ``true``, also contains space-seperated int/strings as labels.
+      - ``true Standing Running Walking Badminton``
+    * - ``@data``
+      - Marks the beginning of data.
+      - The data begins from the next line.
+      - \-
+
+Instructions
+------------
+This section provides full set of instructions to create your format specification for your dataset that is compatible with ``sktime``.
+Remember that this begins with the assumption that you have the dataset readily available in
+expected `format <https://github.com/alan-turing-institute/sktime/blob/main/examples/loading_data.ipynb>`_.
+
+Few points to keep in mind while creating the dataset:
+
+  1. The general order of identifiers **does not** matter with the exception that ``@data`` should be the last string identifier.
+  2. For lines containing an identifier, it **must** begin with it.
+  3. The **only** place to give a space is between identifier and its corresponding value.
```

Comment:
The **only** place a space is allowed is between an identifier and its corresponding value.